libraries/Microsoft.Bot.Builder.Expressions.Parser/Expression.g4

```diff
@@ -32,6 +32,8 @@ fragment DIGIT : [0-9];
 
 NUMBER : DIGIT + ( '.' DIGIT +)? ;
 
+REGEX : '/' .*? '/' [gmi]?;
```

Comment:
Given we want to use or unify regex in dialog recognizer, probably we should use ""